### DISCUSSION TITLE
More maintainers

### DISCUSSION THREAD

**Discussion by hamnis** [circe/circe#2165] [General]
Is there any way to add more maintainers to this project? It seems like there are little progress on issues, and PRs keep piling up.

I understand that OSS burnout is a real thing, and I have experienced that myself. I hope this is not the case here.

I call upon the community to step up to help if there is more that can be done to help offload the maintainers.

**Comment by zmccoy** [comment]
Hi @hamnis, we're open to get folk in the right place on projects they're interested in helping maintain.  Happy to have chats with folk who want to help out of course.

**Reply by hamnis** [reply]
After @zmccoy has now left there is only one maintainer here: @zarthross 
They seem too busy to do much here, so maybe someone else can step up?
I volunteer, but maybe someone else is also interested in helping out?

This library is too important for it to die.

**Reply by zarthross** [reply]
I should officially welcome @hamnis to the circe maintainers team!  

If anyone else is interested in helping out feel free to let us know here.  If there is a specific repo you are interested in, please let us know.
